src/utils/log.ts

```diff
@@ -1,4 +1,4 @@
-export function warn(content: string | (() => void)) {
+export function warn(content: any | (() => void)) {
```

Comment:
we're calling `warn` with an array of `string | Element` above, so I just put `any` here. It shouldn't cause problems since `console.warn` takes `any`...